### DISCUSSION TITLE
Resize of modal causing weird scroll

### DISCUSSION THREAD

**Discussion by hawwokitty** [React95/React95#465] [Q&A] [answered]
Hello! I don't know if it's just happening to me, but when i have the "resize" class on a modal and the content overflows, the scrollbar appears on the side of the entire modal, not just the content of it, so if you start to scroll then the titlebar will disappear with the scroll, if that makes sense.
Test it out here:
[portfolio link](https://hawwokitty.github.io/my-portfolio/)
If you open "Streaming" by double clicking, and resize the modal to be shorter than the content itself, then scroll, you'll see what i mean 🤔 

[Here's the link to my code ](https://github.com/hawwokitty/my-portfolio/tree/main)

Is there a way to fix this? Did i add the resize class wrong? i just set the modals class to resize, like this
`className="resize"`
and then i have this in the css file:
`
.resize {
  resize: both;
  overflow: auto;
}
`

**Comment by ggdaltoso** [comment, marked as answer]
I'm not sure if this is the behavior you're looking for but I managed to get the resizable modal to be like this:


https://github.com/user-attachments/assets/c71e5058-abe5-4d7d-94c8-c94564507a73

By changing [StreamingComp.jsx#L219-L232](https://github.com/hawwokitty/my-portfolio/blob/main/src/components/StreamingComp.jsx#L219-L232) (in addition the current implementation).

Adding an `overflow` prop to the Frame component should do the trick.

```jsx
<Frame
    key="streaming-frame-3"
    w="100%"
    h="100%"
    bgColor="$material"
    boxShadow="$out"
    padding="$4"
    overflow="auto"
  >
	<Frame
		key="streaming-frame-4"
		h="100%"
		bgColor="white"
		boxShadow="$in"
		overflow="auto"
	>
```

Also, [Frame has a `position` prop where you can set it to `"relative"`](https://github.com/hawwokitty/my-portfolio/blob/main/src/components/StreamingComp.jsx#L203), if you want to

**Reply by hawwokitty** [reply]
Yes!! Yhat's exactly what I meant, thank you! 😄
